Packages/vcs/Lib/VTKPlots.py

```diff
@@ -1171,8 +1158,8 @@ def plotContinents(self,x1,x2,y1,y2,projection,wrap,tmpl):
           contActor.GetProperty().SetColor(0.,0.,0.)
       else:
           geo=None
-      contMapper.SetResolveCoincidentTopologyPolygonOffsetParameters(1, 1)
-      contMapper.SetResolveCoincidentTopologyToPolygonOffset()
+      #contMapper.SetResolveCoincidentTopologyPolygonOffsetParameters(1, 1)
```

Comment:
yes you added this one to get continents going but now it makes them disappear , any idea why? Didn't want to take it out completely before we are sure we don't need it.